content/en/security/application_security/software_composition_analysis/_index.md

```diff
@@ -52,40 +52,48 @@ Check the ASM Compatibility for each ASM product to see if your service is suppo
+The [Vulnerability Explorer][3] shows a complete list of the open source libraries detected by Datadog SCA and reports security vulnerabilities associated with them.
 
-Datadog SCA leverages two techniques to analyze your services: 
+Datadog SCA leverages two techniques to analyze your services:
 
 - Static code analysis in your repositories (static point of view)
 - Runtime analysis in your deployed services (runtime point of view)
 
 Combining both techniques monitors open source libraries end-to-end, from the code repository commit (static point of view), to the applications running in production (runtime point of view).
 
-To switch to the code repository commit point of view, select **Static**. The static view shows vulnerabilities from the _source code_ in your repositories. 
+To switch to the code repository commit point of view, select **Static**. The static view shows vulnerabilities from the _source code_ in your repositories.
 
 To switch to the _real-time_ point of view for the applications already running, select **Runtime**. The runtime view is the live view of the services monitored by Datadog.
 
 {{< img src="/security/application_security/software_composition_analysis/asm_sca_vulnerabilities.png" alt="Software Composition Analysis (SCA) explorer page showing vulnerabilities sorted by static or runtime." style="width:100%;" >}}
 
-Select a specific vulnerability to see its details, including the affected services, severity breakdown score, and recommended remediation steps. 
+Select a specific vulnerability to see its details, including the affected services, severity breakdown score, and recommended remediation steps.
 
 On the Details Explorer for a vulnerability, you can view impacted infrastructure. This view gives you better insights to your overall attack exposure.
 
-Within ASM, the vulnerability severity base score is modified using existing attacks and the business sensitivity of the environment where the vulnerability is detected. For example, if no production environment is detected, the severity is reduced.
+## Datadog Severity Score
+
+Each vulnerability has a defined base severity score. To help prioritize remediation, Datadog provides an enhanced severity score (known as "Datadog Severity Score") using existing attacks
+and the business sensitivity of the environment where the vulnerability is detected.
+
+For example, if no production environment is detected, the severity is reduced.
 
 The adjusted vulnerability score includes the full context of each service:
 
 - The original vulnerability severity
-- Evidence of suspicious requests
+- Evidence of suspicious requests or attacks
 - Sensitive or internet-exposed environments
+- Risk of a potential successful exploit
+
+Datadog shows how the base CVSS score is updated into a Datadog Severity Score based on your execution environment.
```

Comment:
not only the execution environment but also the exploitability of the CVE:
 * is there proof this CVE has been exploited ? e.g. from https://www.exploit-db.com/
 * is there a high probability this CVE would be exploited ? from [EPSS](https://www.first.org/epss/)